site/src/pages/UserSettingsPage/SchedulePage/ScheduleForm.tsx

```diff
@@ -79,6 +79,7 @@ export const ScheduleForm: FC<ScheduleFormProps> = ({
 			},
 		});
 	const getFieldHelpers = getFormHelpers<ScheduleFormValues>(form, submitError);
+	const browserLocale = navigator.language || "en-US";
```

Comment:
Disappointed we have to use _language_ as an approximation 😔. I prefer to use English US for text and UI language but Finland is used for numbers, time and currency. Yet this is not available to the browser (AFAICT).